src/coreclr/src/jit/target.h

```diff
@@ -3,8 +3,8 @@
 // See the LICENSE file in the project root for more information.
 
 /*****************************************************************************/
-#ifndef _TARGET_H_
```

Comment:
You should leave this file alone

Reply:
@lpereira Pointed out that the use of identifiers with a leading underscore followed by a capital letter is reserved for the compiler host implementation in C/C++.  I fixed this now, because it showed up in my search.